ansible/kubernetes.yml

```diff
@@ -9,12 +9,26 @@
       listen:
         - systemd_read_configs
   post_tasks:
+    - name: ansible_debug
+        This task is executed by vagrant provision --provision-with ansible-debug
+        this task is only an example, it can be replace by your own debug task
+        or other tasks with ansible_debug tag can be created
+      become: true
+      changed_when: false
+      shell: |
+        set -e ; \
+        echo $0 ; \
+        whoami ; \
+        echo $PATH
+      register: ansible_debug_output
```

Comment:
Then, what are you doing with this variable? Are you writing it somewhere as output?

Reply:
@ferrarimarco this is an example, but you can read the variable in the -vv log, so in principle you do not need to print (this is how I've used it)...
we can add a printout for the clarity of the example (any suggestion?) or I can add the instructions to read the "-vv" log in the task description

Reply:
Right! I forgot about the fact that we add -vv sometimes.

I was thinking more about this: https://docs.ansible.com/ansible/latest/modules/debug_module.html

Reply:
ok I've added a second debug task to msg the output of the first one :-)